packages/docs/validator-guide/attestation-service.md

```diff
@@ -244,18 +265,46 @@ celocli account:get-metadata $CELO_VALIDATOR_RG_ADDRESS
 
 ## Running the Attestation Service
 
-Before running the attestation service, ensure that your local node is fully synced and that the metadata has been registered.
+There is a slight difference in starting up the service depending on the keystore management option (keystore files, full node) used, and both of these are described below.
+
+Before running the Attestation Service using either method, ensure that the metadata has been registered.
+
+Both of the following options for running the Attestation Service use `--network host` to access a local database (only works on Linux), and listen for connections on port 80.
+
+They both assume that all of the configuration options needed have been added to the config file located under `$CONFIG` which Docker will process. Alternatively, you can pass the config file for the service to read on startup using `-e CONFIG=<docker-path-to-config-file>`, and other environment variables directly by adding arguments of the form `-e DATABASE_URL=$DATABASE_URL`.
+
+For both options below, set the `TAG` environment variable to determine which image to install. Use `attestation-service-mainnet` for the latest image suitable for mainnet (as below), `attestation-service-baklava` for the latest image suitable for baklava, or specify a specific build as given in the release notes linked above.
+
+### (recommended) Running the Attestation Service v1.3.0+
+
+The main difference between the old method of running the Attestation Service alongside a Celo full node is that you will need to ensure that Docker can access the directory containing the `keystore` which stores the `CELO_ATTESTATION_SIGNER_ADDRESS`'s private key as an encrypted file.
+
+To do this, you need to map the directory containing the `keystore` to the Attestation Service's Docker volume and set the environment variable `ATTESTATION_SIGNER_KEYSTORE_DIRPATH` to the path to this directory relative to the Docker container. You will also need to set the environment variable `ATTESTATION_SIGNER_KEYSTORE_PASSPHRASE` to the password used during the creation of the `CELO_ATTESTATION_SIGNER_ADDRESS`.
+
+Additionally, ensure that the `CELO_PROVIDER` environment variable points to a Celo node that can be used to instantiate ContractKit; this can be a separate full node (either one you run or forno) or, depending on your configuration, your validator proxy node.
```

Comment:
We should not be recommending forno as primary use for production services.

Instead, we can refer to other node service providers. Ultimately, we should have a docs page with all the node-as-a-service options, but for now we can say:
`either one you run or an endpoint to a node service provider like Figment Datahub`

Ideally, we'd want to recommend leveraging the validator proxy node for reading chain data, but if there are limitations that make this unfeasible for most, let's chat more about that.

Reply:
Note: discussed on slack -- added a note about Figment now; will add further docs changes regarding backup nodes in a later PR.